### ISSUE TITLE
About the clean list

### ISSUE THREAD

**Issue by laycoding** [deepinsight/insightface#65]
Can you share the MID&&n00xxxx classmap corresponding to labels in ms1m and vggface2 .rec files respectively? I want to merge these two dataset through the labels(the number of same celebrities in them is enormous) try to find the relationship between the depth and width of the dataset.
However, there are extra data in your rec files,  i don't konw which identity in original dataset the label mapping to.As you can see: the first person in the rec file and clean list.
![image](https://user-images.githubusercontent.com/17764091/36707686-50a8502a-1baa-11e8-892f-95db29a40c66.png)
![image](https://user-images.githubusercontent.com/17764091/36707701-69c9cc46-1baa-11e8-9afb-60b1536e28a6.png)
it's definitely not the same person
I will be very appreciate if you can share them.

**Comment by nttstar** [comment]
I can not give such information as it is random. See `src/data/face2rec2.py` to know how we generate rec files. 
Datasets merging can be done by comparing centre-vectors, see `src/data/dataset_merge.py`

**Comment by laycoding** [comment]
谢谢

**Comment by guoxf3z** [comment]
Hi @nttstar , could you provide any examples of how to use dataset_merge.py? 
What I am wondering is, should I set --include /path/to/ms1m --exclude /path/to/vggface2 or they should all be include? I don't know which model I can use for this task, a pretrained r34 model that you released or something else called softmax-50?

Thank you

**Comment by test4fest** [comment]
@guoxf3z 
Which pre-trainded model did you used for merging these two datasets?
can you provide complete description how to use dataset_merge.py file?
Thanks
